app/(dashboard)/layout.tsx

```diff
@@ -17,28 +21,65 @@ const Logo = () => (
   </span>
 );
 
+const NavLink = ({
+  href,
+  children,
+}: {
+  href: string;
+  children: React.ReactNode;
+}) => {
+  const pathname = usePathname();
+  const isActive = pathname === href;
+
+  return (
+    <Link
+      href={href}
+      className={cn(
+        "px-3 py-2 text-sm font-medium rounded-md",
+        isActive
+          ? "bg-gray-100 text-gray-900"
+          : "text-gray-500 hover:text-gray-900 hover:bg-gray-50",
+      )}
+    >
+      {children}
+    </Link>
+  );
+};
+
 export default function Layout({ children }: { children: React.ReactNode }) {
   return (
     <ClerkProvider>
       <header className="border-b border-gray-200">
-        <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 py-4 flex justify-between items-center">
-          <SignedIn>
-            <Link href="/dashboard" className="flex items-center">
-              <Logo />
-            </Link>
-          </SignedIn>
-          <SignedOut>
-            <Link href="/" className="flex items-center">
-              <Logo />
-            </Link>
-          </SignedOut>
-          <div className="flex items-center space-x-4">
-            <SignedOut>
-              <SignInButton />
-            </SignedOut>
-            <SignedIn>
-              <UserButton />
-            </SignedIn>
+        <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 py-4">
+          <div className="flex justify-between items-center">
+            <div className="flex items-center space-x-8">
+              <SignedIn>
+                <Link href="/dashboard" className="flex items-center">
+                  <Logo />
+                </Link>
+              </SignedIn>
+              <SignedOut>
+                <Link href="/" className="flex items-center">
+                  <Logo />
+                </Link>
+              </SignedOut>
+
+              <SignedIn>
+                <nav className="flex items-center space-x-4">
+                  <NavLink href="/dashboard">Dashboard</NavLink>
+                  <NavLink href="/dashboard/setup">Setup</NavLink>
```

Comment:
The path for this should be `/projects` (to match the table name), and keep it simple.